rules/avoid-deleting-records-by-flagging-them-as-isdeleted/rule.md

```diff
@@ -1,25 +1,23 @@
 ---
 type: rule
-archivedreason: 
-title: Data - Do you avoid deleting records by flagging them as IsDeleted (aka Soft Delete)?
-guid: 56fff82d-9ed7-4aee-ab22-68e4c61fe162
+title: Data - Do you avoid deleting records by flagging them as IsDeleted (aka
+  Soft Delete)?
```

Comment:
The title was in a single line, and now it is in multiple lines. If this is done by the Rules CMS editor, it's probably fine but if this was because of a local editor, we probably should revert back to a single line.